test/services/fileStorage/index.test.js

```diff
@@ -33,6 +33,10 @@ describe('fileStorage service', () => {
 		assert.ok(app.service('files'));
 	});
 
+	it('registered the thumbnail service', () => {
```

Comment:
see news service tests for more ideas of writing tests

Reply:
We need to first update the filepreview service and deploy it
https://github.com/schul-cloud/filepreview/pull/2